cfpb/complaint
# License exceptions

Adds TERMS.md file, license-exceptions module + a list of license exceptions. These are required by the [open source checklist](https://github.com/cfpb/open-source-project-template/blob/master/opensource-checklist.md)
## Additions
- `TERMS.md`
- `license-exceptions.js` for updating the TERMS Exceptions list. To update manually anytime new packages have been added, run `node src/static/js/license-exceptions.js`
## Testing
- check out branch
- run `node src/static/js/license-exceptions.js` to update the license exceptions list in the TERMS.md file
## Review
- @virginiacc 
## Todos
- update license-exceptions package to output version numbers. seeming duplicates in the output list are due to different versions.
## Checklist
